go.mod

```diff
@@ -28,7 +29,8 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
 	github.com/spf13/viper v1.3.1
-	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80 // indirect
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
```

Comment:
Is there a released version we could use ?

Reply:
I'll look.

Reply:
No. Or, kind of, but using it ends up with even more downgrades.